libsabergen/sabergen.py

```diff
@@ -11,12 +11,23 @@
 import librosa.display as disp
 
 class beatSaberSong(object):
-    def __init__(self, song_path, debug):
-        self.song_path = song_path
+    def __init__(self, debug=False):
         self.debug = debug
+        self.y = None
+        self.sr = None
+        self.audio_path = None
+
+    def load(self, audio_path):
+        "Loads the audio file at the specified path. Converts to ogg if necessary."
+        self.audio_path = audio_path
+
+        if not self.is_ogg(audio_path):
+            self.audio_path = self.convert_to_ogg(audio_path)
+
+        self.y, self.sr = librosa.load(self.audio_path)
 
     def is_ogg(self, song_path):
-        """ Using the magic number of a file, determine if this is an .ogg file. """
+        "Using the magic number of a file, determine if this is an .ogg file."
```

Comment:
Out of curiosity, why stray from PEP on docstrings?

Reply:
I wasn't aware PEP8 required triple quotes for docstrings - I was under the impression you used triple quotes for multi line and single quotes for one line. `pylint` didn't report an issue so, if I'm wrong, we should update that.

Reply:
You are indeed correct [according to PEP257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings). I'll update that before landing.

Reply:
https://www.python.org/dev/peps/pep-0008/#id33
References the PEP 257. Seems to call out the single line case and the only requirement is to keep """ so I guess string things are not in place here. :shrug: More of a curious thing, not really concerned about it if it still shows up as a docstring in ides